### ISSUE TITLE
🔥 Danger Rules 🔥

### ISSUE THREAD

**Issue by squarefrog** [RocketFanOrg/RocketFanApp#5]
Now that [Danger](https://danger.systems) is up and running, I think we should solidify more rules. Here are my suggestions:
 
- [ ] Provide a summary in a Pull Request
- [ ] If README is touched, spell check using `danger-prose`
- [ ] Require no merge commits
- [ ] Run SwiftLint
- [ ] Run SwiftFormat
- [ ] Suggest adding tests if lines of code changed > 30?

Lets add to this as we develop. We can see where the sticking points are and add changes incrementally.

**Comment by RJ-Clegg** [comment]
Agreed. The suggestions look good to get us started - from there we can just build / change as we need as the project grows.

**Comment by squarefrog** [comment]
Ok, so I think the only two were interested in now would be swift lint and format. I’ll figure that out then we’ll do the rest as we see fit.
